app/javascript/src/components/Common/Menu.jsx

```diff
@@ -4,13 +4,11 @@ import { Search, Plus, Settings } from "neetoicons";
 import { Typography } from "neetoui";
 import { MenuBar } from "neetoui/layouts";
 
-import { MENU_BAR_BLOCKS } from "./constants";
-
-const Menu = ({ showMenu }) => {
+const Menu = ({ showMenu, menuBarBlocks }) => {
```

Comment:
Why was this moved to the `Common` namespace? Are we using this component at multiple places within the app?

Reply:
Yes, I have used the Menu component in both Notes and Contacts page.